package.json

```diff
@@ -191,6 +190,9 @@
     "yargs": "^16.1.1",
     "yargs-parser": "^20.2.4"
   },
+  "resolutions": {
+    "v8-to-istanbul": "https://github.com/connorjclark/v8-to-istanbul.git#06abc4c4"
```

Comment:
waiting on https://github.com/bcoe/c8/pull/273

Reply:
@connorjclark I apologize for the slow turnaround, was a little buried catching up after holidays.

Let me know if you bump into any issues.